modules/proc-customizing-the-web-console.adoc

```diff
@@ -9,6 +9,11 @@ RHDH supports the following theme modes:
 * Dark theme
 * Auto
 
+[NOTE]
+====
+In the {product} application, theme configurations are used to change the look and feel of different UI components. So, you might notice changes in different UI components, such as buttons, tabs, sidebars, cards, and tables along with some changes in background color and font used on the {product-very-short} pages. 
```

Comment:
```suggestion
In {product-short}, theme configurations are used to change the look and feel of different UI components. So, you might notice changes in different UI components, such as buttons, tabs, sidebars, cards, and tables along with some changes in background color and font used on the {product-very-short} pages. 
```

Reply:
@shipsing WRT to the position of the Note, it appears between the introduction and the procedure to change the theme. Perhaps it could be placed immediately after the heading or after the procedure?

Reply:
@Gerry-Forde, I incorporated your suggestion and placed the note just after the heading to notify readers about this look and feel change in UI.